src/stores/datastore.js

```diff
@@ -1,103 +1,26 @@
 'use strict'
 
-const Lock = require('lock')
-const stream = require('stream')
-const bl = require('bl')
-const Block = require('ipfs-block')
+// TODO: This may end up being configurable
+// TODO: This should belong to the `fs` implementation
```

Comment:
This is the code that was here before in datastore-legacy.js I did not touch it just rename it